sympy/sets/fancysets.py

```diff
@@ -443,3 +444,126 @@ def _sup(self):
     @property
     def _boundary(self):
         return self
+
+
+def as_r_theta(z):
```

Comment:
We don't need this function, we already have `arg` [1] and `Abs` [2] as a part of elementary functions.

[1]: http://docs.sympy.org/dev/modules/functions/elementary.html#arg
[2]: http://docs.sympy.org/dev/modules/functions/elementary.html#abs